File: example/typescript/SomeModule/Api/Interfaces.ts
interface SomeInterface {
    /**
     * @throws { SomeException }
     * @throws { Some2Exception }
     */
    someCommand(id: SomeId, amount: number, c: HandlerContext): void

    /**
     * @throws { SomeException }
     */
    someQuery(id: SomeId, c: HandlerContext): SomeClass
}

File: example/kotlin/src/main/java/pl/bratek20/somemodule/api/Interfaces.kt
package pl.bratek20.somemodule.api

interface SomeInterface {
    @Throws(
        SomeException::class,
        Some2Exception::class
    )
    fun someCommand(id: SomeId, amount: Int): Unit

    @Throws(
        SomeException::class
    )
    fun someQuery(id: SomeId): SomeClass
}